CHANGELOG/CHANGELOG-1.14.md

```diff
@@ -1,5 +1,23 @@
 # Changelog 1.14
 
+## 1.14.2 - 2024-01-19
+
+### Changed
+
+- Enable IPv4/IPv6 forwarding on demand automatically to eliminate the need for user intervention or dependencies on other components. ([#5833](https://github.com/antrea-io/antrea/pull/5833), [@tnqn])
+
+### Fixed
+
+- Store NetworkPolicy in filesystem as fallback data source to let antrea-agent fallback to use the files if it can't connect to antrea-controller on startup. ([#5739](https://github.com/antrea-io/antrea/pull/5739), [@tnqn])
+- Support Local ExternalTrafficPolicy for Services with ExternalIPs when Antrea proxyAll mode is enabled. ([#5795](https://github.com/antrea-io/antrea/pull/5795), [@tnqn])
+- Enable Pod network after realizing initial NetworkPolicies to avoid traffic from/to Pods bypassing NetworkPolicy when antrea-agent restarts. ([#5777](https://github.com/antrea-io/antrea/pull/5777), [@tnqn])
+- Fix Clean-AntreaNetwork.ps1 invocation in Prepare-AntreaAgent.ps1 for containerized OVS on Windows. ([#5859](https://github.com/antrea-io/antrea/pull/5859), [@antoninbas])
+- Add missing space for kubelet args in Prepare-Node.ps1 to resolve kubelet not starting issue on Windows. ([#5858](https://github.com/antrea-io/antrea/pull/5858), [@antoninbas])
```

Comment:
Add missing space to kubelet args in Prepare-Node.ps1 so that kubelet can start successfully on Windows.

Reply:
Updated.